x-pack/plugins/security_solution/public/management/components/endpoint_response_actions_list/response_actions_log.tsx

```diff
@@ -115,8 +115,12 @@ export const ResponseActionsLog = memo<
     } = useGetEndpointActionList(
       {
         ...queryParams,
-        startDate: isFlyout ? dateRangePickerState.startDate : startDateFromUrl,
-        endDate: isFlyout ? dateRangePickerState.endDate : endDateFromUrl,
+        startDate: isFlyout
+          ? dateRangePickerState.startDate
+          : startDateFromUrl ?? DEFAULT_DATE_RANGE_OPTIONS.startDate,
+        endDate: isFlyout
+          ? dateRangePickerState.endDate
+          : endDateFromUrl ?? DEFAULT_DATE_RANGE_OPTIONS.endDate,
```

Comment:
So here we default to the "last 24 hours" when no date range is set on URL params on the page.